blocks/header/__navigation/header__navigation.css

```diff
@@ -0,0 +1,39 @@
+.header__navigation
+{
+    margin-top: 7px;
+    margin-left: 863px;

```

Comment:
I think we should consider using `display: flex` to have prettier and more modern approach.